LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
"Adding a licence" should be a separate PR

Reply:
#334